orc8r/cloud/helm/orc8r/charts/secrets/values.yaml

```diff
@@ -10,12 +10,59 @@ create: true
 
 # orc8r secret configuration
 secret:
-  # base directory holding cert secrets
-  certs: .secrets/certs
+  certs:
+    enabled: false
+  # files:
+  #   #Controller Crt
+  #   controller.crt: |-
+  #     ...
+  #     ...
+  #   #Controller Key
+  #   controller.key: |-
+  #     ...
+  #     ...
+  #   #rootCA
+  #   rootca: |-
+  #     ...
+  #     ...
+  #   #NMS Operator Crt
+  #   admin_operator.pem: |-
+  #     ...
+  #     ...
+  #   #NMS Operator Key
+  #   admin_operator.key.pem: |-
+  #     ...
+  #     ...
+  #   #Fluentd Crt
+  #   fluentd.pem: |-
+  #     ...
+  #     ...
+  #   #Fluentd Key
+  #   fluentd.key: |-
+  #     ...
+  #     ...
+
   # base directory holding config secrets.
-  configs: .secrets/configs
-  # base directory holding envdir secrets.
-  envdir: .secrets/envdir
+  configs:
+    enabled: false
+  # orc8r:
+  #   metricsd: |-
```

Comment:
Can you uncomment this guy, and see if helm will let you replace the `orc8r-` prefixes to all the URL values with `{{ .Release.Name }}`? This will remain the same for 99% of deployments

Reply:
@xjtian  fixed .. we can't use {{ .Release.Name }} here. its not supported when we use range.  if its just one value insted of range then we can use tpl function to render {{ .Release.Name }} to template .  Please let me know if it will be only metrics.yaml under orc8r-secrets-configs-orc8r ?

Reply:
oh yeah, right of course we can't use template variables in a values file.

lgtm